packages/terra-tabs/src/Tabs.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import ContentContainer from 'terra-content-container';
-import ResponsiveElement from 'terra-responsive-element';
+import ResponsiveElement, { DependentViewport } from 'terra-responsive-element';
```

Comment:
I think the `DependentViewport` constant makes the code more challenging to understand and doesn't follow a consistent export strategy. There are only two options `window` and `parent`. I'd recommend just using the string literals. `window`, `parent`. 

I don't personally think keying objects in this context lowers the chances of typos/errors or adds enough reusability to adjust for the added complexity.

Reply:
> I don't personally think keying objects in this context lowers the chances of typos/errors or adds enough reusability to adjust for the added complexity.

It's been a grey area for me as to when to use it and when not to but this makes sense. I have [removed](https://github.com/cerner/terra-framework/pull/968/commits/4264fffd83d8119e411ddf2b7d1f7d303bf63c76) it.